tests/drivers/build_all/input/it82xx2_evb.overlay

```diff
@@ -6,25 +6,27 @@
 
 /delete-node/ &kbd;
 
-it8801_i2c: &i2c4 {
+&i2c4 {
 	status = "okay";
+
+	it8801_mfd: it8801@38 {
+		compatible = "ite,it8801-mfd";
+		reg = <0x38>;
+		irq-gpios = <&gpioa 1 0>;
+	};
 };
 
 #include <ite/it8801-common-cfg.dtsi>
 
-&it8801_mfd {
-	irq-gpios = <&gpioa 1 0>;
-
-	ioex_it8801_kbd: it8801_kbd@40  {
-		status = "okay";
-		/* GP00, GP01, GP21, GP22, GP23 can be configured as alternate KSO pins */
-		mfdctrl = <&kso18_gp01_default &kso20_gp23_default>;
-		kso-mapping = <0 1 20 3 4 5 6 17 18 16 15 11 12>;
-		row-size = <8>;
-		col-size = <13>;
+&ioex_it8801_kbd  {
```

Comment:
Nit: I accidentally left a space earlier.

Reply:
oh thank you, fixed